interfaces/acados_template/acados_template/generate_c_code_implicit_ode.py

```diff
@@ -95,11 +95,18 @@ def generate_c_code_implicit_ode( model, opts ):
         fun_name = model_name + '_impl_dae_fun_jac_x_xdot_z'
         impl_dae_fun_jac_x_xdot = Function(fun_name, [x, xdot, u, z, p], [f_impl, jac_x, jac_xdot, jac_z])
 
-        fun_name = model_name + '_impl_dae_jac_x_xdot_u'
-        impl_dae_jac_x_xdot_u = Function(fun_name, [x, xdot, u, z, p], [jac_x, jac_xdot, jac_u, jac_z])
+        # fun_name = model_name + '_impl_dae_fun_jac_x_xdot_z'
+        # impl_dae_fun_jac_x_xdot = Function(fun_name, [x, xdot, u, z, p], [f_impl, jac_x, jac_xdot, jac_z])
+
+        # fun_name = model_name + '_impl_dae_jac_x_xdot_u'
+        # impl_dae_jac_x_xdot_u = Function(fun_name, [x, xdot, u, z, p], [jac_x, jac_xdot, jac_u, jac_z])
         
         fun_name = model_name + '_impl_dae_fun_jac_x_xdot_u_z'
         impl_dae_fun_jac_x_xdot_u = Function(fun_name, [x, xdot, u, z, p], [f_impl, jac_x, jac_xdot, jac_u])
```

Comment:
I guess `_z` should be removed from the name, since `jac_z` is not part of output

Reply:
fixed with e2421d.